docker-compose.yml

```diff
@@ -826,6 +827,9 @@ services:
       SubmissionServiceConfig__MeasureNames__2__Url: http://www.cdc.gov/nhsn/fhirportal/dqm/ig/Measure/NHSNAcuteCareHospitalDailyInitialPopulation
       SubmissionServiceConfig__MeasureNames__2__MeasureId: NHSNAcuteCareHospitalDailyInitialPopulation
       SubmissionServiceConfig__MeasureNames__2__ShortName: ACHD
+      SubmissionServiceConfig__MeasureNames__3__Url: http://www.cdc.gov/nhsn/fhirportal/dqm/ig/Measure/NHSNdQMAcuteCareHospitalInitialPopulation
+      SubmissionServiceConfig__MeasureNames__3__MeasureId: NHSNdQMAcuteCareHospitalInitialPopulation
+      SubmissionServiceConfig__MeasureNames__3__ShortName: ACH
```

Comment:
This is the old version of ACH monthly.  We should prefer the new name/ID/URL (already present at index 0).